src/Honeybee_ExportEPC.py

```diff
@@ -127,18 +140,48 @@ def main(HBZones):
     # return surface data
     return srfData
     # done!
-
-
+#print srfData
 srfTypeDict = {0: "Wall", 1 : "Roof", 2: "Floor", 5 : "Glazing"}
 directionDict = {0: "N", 1 : "NW", 2: "W", 3: "SW", 4 : "S", 5 : "SE", 6 : "E", 7 : "NE"}
 
+
```

Comment:
@pchopson I checked the code again and now I'm confused about how you really need the data. Why are you adding up the area of surfaces with different constructions back together? Don't you need the area separated for each construction? If that is not the case I can simplify the initial dictionary so you don't have to do it down there again.